src/components/Infobar/infobar.scss

```diff
@@ -2,13 +2,26 @@
   background: #f5f5f5;
   height: 50px;
   margin-bottom: 20px;
-  .listInline {
+  font-weight: bold;
+  font-family: 'Open Sans', sans-serif;
+  color: #3c3d41;
+
+  .mfound {
     display: inline-block;
-    li {
-      display: inline-block;
-    }
+    float: left;
+  }
+
+  .filters {
+    float: right;
+    width: auto;
```

Comment:
When you float elements they will have auto width by default. So, you can remove width: auto